code/modules/cargo/bounties/slime.dm

```diff
@@ -3,7 +3,7 @@
 
 /datum/bounty/item/slime/New()
 	..()
-	description = "Nanotrasen's science lead is hunting for the rare and exotic [name]. A bounty has been offered for finding it."
+	description = "Horizons science lead is hunting for the rare and exotic [name]. A bounty has been offered for finding it."
```

Comment:
Is there a missing apostrophe?

*I actually cant tell please help*

Reply:
Yes.